xbmc/utils/BooleanLogic.h

```diff
@@ -86,16 +86,13 @@ class CBooleanLogicOperation : public IXmlDeserializable
   CBooleanLogicValues m_values;
 };
 
-class CBooleanLogic : public IXmlDeserializable
+class CBooleanLogic
```

Comment:
Why does this not have any destructor?

Reply:
You're right, this one should have a `protected` defaulted destructor.